Sources/Reg/Reg.swift

```diff
@@ -47,6 +47,33 @@ public struct Regex: ExpressibleByStringLiteral {
         return String(string[matchRange])
     }
 
+    // TODO: add string extensions for each method
+    // TODO: add tests
+    public func replacingOccurrences(in string: String, with replacement: String = "") -> String? {
```

Comment:
Instead of using a default param, maybe just alias it w/ 
```swift
    func removingOccurrences(in string: String) -> String? {
        return replacingOccurrences(in: string, with: "")
    }
```